### ISSUE TITLE
1.6.1 still failing to delete nodes randomly

### ISSUE THREAD

**Issue by xzy3** [aws/aws-parallelcluster#644]
v1.6.1, Centos 7, GridEngine, custom AMI

Found 3 zombie nodes with jobs scheduled on them.

```bash
qdel -f <job_id>
qconf -de <node name>
```
removed the nodes successfully.

/var/log/sqswatcher
```
2018-11-13 17:15:30,947 INFO [sqswatcher:pollQueue] eventType=autoscaling:EC2_INSTANCE_TERMINATE
2018-11-13 17:15:30,947 INFO [sqswatcher:pollQueue] instanceId=i-045470a4f016fc562
2018-11-13 17:15:31,168 INFO [sge:removeHost] Removing ip-*********
root@ip-**************** modified "all.q" in cluster queue list
root@ip-**************** modified "@allhosts" in host group list
root@ip-**************** removed "ip-*************" from administrative host list
Host object "ip-1**************" is still referenced in cluster queue "all.q".
2018-11-13 17:15:31,219 ERROR [sge:__runSgeCommand] Failed to run ['/opt/sge/bin/lx-amd64/qconf', '-de', 'ip-**************']
```

**Comment by sean-smith** [comment]
Thanks for the bug report. We're looking into it.

**Comment by lukeseawalker** [comment]
Hi all,
this issue has been fixed by: [aws/aws-parallelcluster-node#98](https://github.com/aws/aws-parallelcluster-node/pull/98)
When all the jobs referencing the terminated instance will be delete, then the host will be removed from the scheduler. We have also plans to change the default SGE configuration in order to allow non-admin users to be able to force delete jobs [aws/aws-parallelcluster-cookbook#259](https://github.com/aws/aws-parallelcluster-cookbook/pull/259)

I'm going to close this case since the patch has been merged in the develop branch.
It will be available in the next version of AWS ParallelCluster.

Please let us know if you have any questions.
